src/QA/QA.Web/ViewModels/WithdrawVacancy/FindVacancyEditModelValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+
+namespace Esfa.Recruit.Qa.Web.ViewModels.WithdrawVacancy
+{
+    public class FindVacancyEditModelValidator : AbstractValidator<FindVacancyEditModel>
+    {
+        public FindVacancyEditModelValidator()
+        {
+            RuleFor(x => x.VacancyReference)
+                .NotEmpty()
+                .WithMessage("Please enter vacancy reference number to continue");
```

Comment:
Should that be "Please enter **a** vacancy reference number"?

Reply:
Probably. This is just a cut and paste from the prototype. It reads better with an 'a'. So I'll add. 👍